tests_integration/test_object.py

```diff
@@ -118,3 +121,55 @@ def test_show_drop_image_repository(runner, test_database, snowflake_session):
     )
     assert result_drop.exit_code == 0, result_drop.output
     assert f"{repo_name} successfully dropped" in result_drop.output
+
+
+@pytest.mark.parametrize(
```

Comment:
`table`, `task`, `warehouse` work when called manually, integration test role *probably* don't have enough permissions

Reply:
I think we should grant create for them and test it, but this will requires extending of our cleanup function. Can be done in separate task.

Reply:
https://snowflakecomputing.atlassian.net/browse/SNOW-1482578